ehrapy/api/preprocessing/_data_imputation.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict, Optional, Tuple, Union
+
+import numpy as np
+from anndata import AnnData
+
+
+class Imputation:
```

Comment:
In general; It would be better to no not have this in this PR, since it's not fully implemented and I really did not expected it here, since this would be something that better fits into quality control.

Reply:
I needed it for the notebook update. But I can add it in another PR that you can then use as your base.

Reply:
IMO, for now its fine, just wanted to state it.